files/image_config/cron.d/logrotate

```diff
@@ -1,2 +1,3 @@
-# Attempt to rotate system logs once per minute
-* * * * * root /usr/sbin/logrotate /etc/logrotate.conf > /dev/null 2>&1
+# Attempt to rotate system logs once every 10 minutes.
+# First kill any logrotate process(es) if they are still running, as they're most likely hung
+*/10 * * * * root /usr/bin/pkill -9 logrotate > /dev/null 2>&1; /usr/sbin/logrotate /etc/logrotate.conf > /dev/null 2>&1
```

Comment:
In the extreme case, pkill will fail. Don't start new `logrotate`.
```suggestion
*/10 * * * * root /usr/bin/pkill -9 logrotate > /dev/null 2>&1 && /usr/sbin/logrotate /etc/logrotate.conf > /dev/null 2>&1
```

Reply:
The problem here is that if there is no process to kill (which is the normal, good case), pkill will return 1, thus the second command will never be run. Therefore, logrotate will never get started because pkill will never succeed because there is no logrotate to kill. The best bet is to ignore the result of pkill, which is what I did.